server/src/main/resources/rules/rules.js

```diff
@@ -1516,7 +1517,8 @@ var Entitlement = {
 
         var requiresConsumerType = context.getAttribute(context.pool, REQUIRES_CONSUMER_TYPE_ATTRIBUTE);
         // skip if the attribtue is not defined.
-        if (requiresConsumerType != null) {
+        log.debug("Checking consumerType:" + context.consumer.type.label);
```

Comment:
non-nacking comment:
Even at the JS-level, we can use the "Checking consumerType: {}", value optimization to avoid unnecessary string concatenation.